.storybook/themes/news.scss

```diff
@@ -1056,6 +1056,26 @@
 				single-chain-children-stack: (
 					gap: var(--global-spacing-6),
 				),
+				small-manual-promo: (
+					components: (
+						media-item: (
+							flex: 1 0 min-content,
+						),
+						image: (
+							display: inline,
+							aspect-ratio: attr(aspect-ratio),
+						),
+						heading: (
+							flex: 2 0 min-content,
+						),
+					),
+				),
+				small-manual-promo-horizontal: (
+					gap: var(--global-spacing-6),
+				),
+				small-manual-promo-vertical: (
+					gap: var(--global-spacing-5),
```

Comment:
Have we called out this discrepancy between the horizontal and vertical gaps in the discrepancies document? This feels like it should be the same gap for both variations

Reply:
Both tokens are called out in the Figma specs:
https://www.figma.com/file/vWVGgZfZVBdSX7ACo3kmjr/Enhanced-Styling-Design-System?node-id=3507%3A32495

I asked Brandon and Kelly about this when developing the block:
https://www.figma.com/file/vWVGgZfZVBdSX7ACo3kmjr?node-id=3507:32495#212870942

I can include this in the Design Discrepancies doc in Confluence. Though it is in Figma, it might be good to note it in the Confluence docs as well.